praw/models/stylesheet.py

```diff
@@ -21,3 +24,18 @@ class Stylesheet(PRAWBase):
     ``stylesheet``          The contents of the stylesheet, as CSS.
     ======================= ===================================================
     """
+
+    def __eq__(self, other: _Stylesheet) -> bool:
+        """Check if two stylesheets are equal."""
+        if isinstance(other, self.__class__):
```

Comment:
Same [type-annotation question](https://github.com/praw-dev/praw/pull/1369#discussion_r396024824) as in #1369.